src/clims/services/extensible.py

```diff
@@ -1,11 +1,56 @@
 from __future__ import absolute_import
 
+import six
 from clims.services.substance import ExtensibleBaseField
 from clims.models import ExtensibleType, ExtensiblePropertyType
 from django.db import transaction
 
 
+class ExtensibleTypeNotRegistered(Exception):
+    pass
+
+
 class ExtensibleService(object):
+    def __init__(self, app):
+        self._app = app
+        self._implementation_cache = dict()
+        self._model_cache = dict()
+
+    def clear_cache(self):
+        self._implementation_cache.clear()
+        self._model_cache.clear()
+
+    def create(self, name, klass, organization, properties=None):
+        """
+        Creates an extensible and saves it to the backend.
+        """
+        if isinstance(klass, six.string_types):
+            klass = self.get_implementation(klass)
+        instance = klass(name=name, organization=organization, _app=self._app)
```

Comment:
They are bound to each organization, but at the moment we register them for all organizations.

The reason for that is that I think we need to discuss implementation details for how to, since things are auto registered, we should bind them to only organization. One idea is to not allow auto registration if there is more than one org in the system.